addons/website_slides/views/website_slides_templates_lesson.xml

```diff
@@ -163,18 +163,18 @@
                     <ul t-if="aside_slide.slide_resource_ids or aside_slide.question_ids" class="list-group px-2 mb-1 list-unstyled">
                         <t t-foreach="aside_slide.slide_resource_ids" t-as="resource">
                             <li class="pl-4">
```

Comment:
we will have empty ```<li class="pl-4">``` (that may be shown depending on style), suggestions:

```suggestion
                            <li class="pl-4" t-if="resource.resource_type == 'url'">
```
or we could filter the foreach with something (untested) like:

```xml
t-foreach="aside_slide.slide_resource_ids.filtered(lambda r: r.resource_type == 'url')"
# or
t-foreach="aside_slide.slide_resource_ids.filtered_domain([('resource_type', '=', 'url')])"
```